src/main/java/core/basesyntax/DrawFIgure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawFIgure {
```

Comment:
The interface name 'DrawFIgure' has a typo and uses a verb which is not recommended. Interface names should be nouns or noun phrases. A better name would be 'FigureDrawer' to reflect its capability.